invenio_github/ext.py

```diff
@@ -69,9 +70,26 @@ def record_serializer(self):
             return import_string(imp)
         return imp
 
+    def cff_yaml(self, release):
+        """Outputs the yaml CFF template from a release object."""
+        return yaml.dump({
+            "cff-version": "1.1.0",
+            "message": "If you use this software, please cite it as below.",
+            "authors": [
+                {"family-names": "Joe",
+                "given-names": "Johnson",
+                "orcid": "https://orcid.org/0000-0000-0000-0000"}
+            ],
+            "title": str(release.title),
+            "version": str(release.model.tag),
+            "date-released": str(release.event.payload['release']['published_at'][:10]
+                             if release.event else '2021-07-28')
+            }, indent=2, default_style='"')
+
```

Comment:
This can go in [`views/github.py`](https://github.com/inveniosoftware/invenio-github/blob/883eb122fb13dc09fc28c1de25bfb435bee2db56/invenio_github/views/github.py#L57-L79), and get annotated with `@blueprint.app_template_filter(...)`.